iaso/tests/api/test_instances.py

```diff
@@ -921,6 +923,32 @@ def test_can_retrieve_instances_in_csv_format(self):
         response = self.client.get(f"/api/instances/?format=csv", headers={"Content-Type": "text/csv"})
         self.assertFileResponse(response, 200, "text/csv; charset=utf-8")
 
+    def test_can_retrieve_submissions_list_in_csv_format(self):
+        """It tests the csv submissions export data"""
+
+        self.yoda.username = "yoda"
+        self.yoda.last_name = "Da"
+        self.yoda.first_name = "Yo"
+        self.yoda.save()
+        self.form_id = self.form_1.id
+
+        self.created_by = self.yoda
```

Comment:
Unless there is a specific reason, could you move the configuration of test objects in `setUpTestData`?